src/ApprenticeAan/SFA.DAS.ApprenticeAan.Api/Controllers/MembersController.cs

```diff
@@ -22,4 +27,31 @@ public async Task<IActionResult> GetMembers([FromQuery] GetMembersRequestModel r
         var response = await _mediator.Send((GetMembersQuery)requestModel, cancellationToken);
         return Ok(response);
     }
+
+
+    [HttpPut("{memberId}")]
+    [ProducesResponseType(StatusCodes.Status204NoContent)]
+    public async Task<IActionResult> UpdateMemberProfileAndPreferences(
+        [FromRoute] Guid memberId,
+        [FromBody] UpdateMemberProfileModel request,
+        CancellationToken cancellationToken)
+    {
+        PatchMemberRequest patchMemberRequest = request.patchMemberRequest;
+
+        JsonPatchDocument<PatchMemberRequest> jsonPatchDocument = new JsonPatchDocument<PatchMemberRequest>();
+        if (patchMemberRequest.RegionId > 0)
+        {
+            jsonPatchDocument.Replace(x => x.RegionId, patchMemberRequest.RegionId);
+        }
+        jsonPatchDocument.Replace(x => x.OrganisationName, patchMemberRequest.OrganisationName);
```

Comment:
Why there is no check for organisation name, this should only be added if it has a value isn't it?

Reply:
Organisation name is nullable string so we shouldn't check for it if there is null then it is also valid value for it

Reply:
Not for Apprentices, this is a required information for Apprentices, see the onboarding journey.

Reply:
Done